Makefile

```diff
@@ -4,27 +4,27 @@ OPAM_LIBDIR=$(DESTDIR)$(shell opam config var lib)
 .PHONY: build release install uninstall clean test doc reindent
 
 release:
-	    jbuilder build @install -j $$(getconf _NPROCESSORS_ONLN)
+	    dune build @install -j $$(getconf _NPROCESSORS_ONLN)
```

Comment:
Should we rename the `jbuild` file to `dune` as well while we're here?

Reply:
We are using the `Jbuild_plugin` - is that available in Dune, too?

Reply:
Yes, but with some caveats. We can't use the Unix module, so Unix.getenv has to be replaced by Sys.getenv.
We have to drop the external calls to ocamlfind and the flags variable (its not needed anyway with the new pps calls).
And then something changed around how many parenthesis you need in dune vs jbuild.
Here is an updated network/dune that should work:
```
(* -*- tuareg -*- *)
let rewriters_ppx = ["ppx_deriving_rpc"]

let coverage_rewriter =
  let is_coverage = try Sys.getenv "BISECT_ENABLE" = "YES" with Not_found -> false in
  if is_coverage then
    "(preprocess (pps (bisect_ppx -conditional)))"
  else
    Printf.sprintf "(preprocess (pps %s))" (String.concat " " rewriters_ppx)

let () = Printf.ksprintf Jbuild_plugin.V1.send {|
(library
 (name xcp_network)
  (public_name xcp.network)
  (flags (:standard -w -39-33))
  (modules (:standard \ network_cli))
  (libraries
   rpclib
    threads
    xcp)
  (wrapped false)
  %s)

(executable
 (name network_cli)
  (modules network_cli)
  (libraries
    cmdliner
    rpclib.cmdliner
    rpclib.markdown
    xcp.network))

(alias
 (name runtest)
   (deps network_cli.exe)
     (action (run ${<})))

|} coverage_rewriter
```

Reply:
This is not just renaming files - Dune demands all kinds of changes that I would rather not do now.

Reply:
Ok, lets do them in a future PR.